Lib/glyphs2ufo/torf.py

```diff
@@ -411,15 +412,25 @@ def build_postscript_name(family_name, style_name):
 def get_weight_code(style_name):
     """Get the appropriate OS/2 weight code for this style."""
 
-    return {
+    weight_code = {
         'Thin': 250,
         'Light': 300,
+        'SemiLight': 350,
+        'DemiLight': 350,
+        '': 400,
+        'Regular': 400,
         'Medium': 500,
+        'DemiBold': 600,
         'SemiBold': 600,
         'Bold': 700,
         'ExtraBold': 800,
         'Black': 900
-    }.get(style_name, 400)
+    }.get(style_name, None)
+    if not weight_code:
+      print('WARNING: Unrecognized style name "%s"' % style_name,
+            file=sys.stderr)
```

Comment:
Could you make printing to sys.stderr the behavior of warn() (defined at the bottom of torf.py), then use warn here?